src/pages/Login.tsx

```diff
@@ -0,0 +1,106 @@
+import {ArrowBackIcon, EmailIcon, Icon} from '@chakra-ui/icons'
+import {
+    Button,
+    Center,
+    FormControl,
+    FormLabel,
+    Heading, HStack,
+    Input,
+    InputGroup,
+    InputRightAddon,
+    Text,
+    useToast,
+    VStack
+} from '@chakra-ui/react'
+import { Field, FieldProps, Form, Formik, FormikProps, FormikValues } from 'formik'
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useFetch } from 'use-http'
+
+export default function Login() {
+    const {post, response} = useFetch()
+    const navigate = useNavigate()
+    const toast = useToast()
+    const [show, setShow] = React.useState(false)
```

Comment:
Chakra UI has a really convenient hook for handling exactly these cases of toggling true/false:
https://chakra-ui.com/docs/styled-system/utility-hooks/use-boolean